File: npanday-it-runner/src/test/resources/ClassLibraryWithWebReference/ClassLibraryWithWebReference/Class1.cs
using System;
using System.Collections.Generic;
using System.Text;
using NUnit.Framework;
using System.Web.Services;

namespace ClassLibraryWithWebReference
{
    [TestFixture]
    public class Class1
    {
        [Test]
        public void Test()
        {
            try
            {
                webService.LegacyCreateFileRequest request = new webService.LegacyCreateFileRequest();
                request.sessionToken = "invalidSession";
                request.fileName = "someFile.txt";

                webService.DiomedeStorageLegacyTransfer svc = new webService.DiomedeStorageLegacyTransfer();

                svc.CreateFile(request);
            }

            catch (System.Web.Services.Protocols.SoapException ex)
            {
                Assert.IsTrue(ex.Message.Contains("Invalid session"));
            }
        }
    }
}

File: plugins/wix-maven-plugin/src/it/IT006/VBTestApp/Class1.vb
Imports NUnit.Framework
Imports VBAppDll

Namespace NUnitTest
    <TestFixture()> _
    Public Class Class1

        <Test()> _
        Public Sub RunTest()
            Dim oHello As New Hello
            Assert.IsTrue(True)
        End Sub

    End Class
End Namespace
